openslides_backend/presenter/check_mediafile_id.py

```diff
@@ -54,8 +54,22 @@ def get_result(self) -> Any:
                     "owner_id",
                     "token",
                     "mimetype",
-                    "used_as_logo_$_in_meeting_id",
-                    "used_as_font_$_in_meeting_id",
+                    "used_as_logo_projector_main_in_meeting_id",
```

Comment:
Please add this fields in a generic way from the new enums.

Reply:
Done.